taskcluster_util/taskcluster_download.py

```diff
@@ -93,19 +93,19 @@ def run(self):
             task_id = self.options.task_id
 
         artifact_downloader = Downloader(connection_options)
-        if self.options.aritfact_name is None and self.options.dest_dir is None:
-            # no artifact_ and dest_dir, then get the latest artifacts list
+        if self.options.aritfact_name is None:
+            # no artifact_name, then get the latest artifacts list
             self.show_latest_artifacts(artifact_downloader, task_id)
-        elif self.options.aritfact_name is None or self.options.dest_dir is None:
-            # if one of artifact_name and dest_dir is specified, return error
-            print('### Please specify the artifact name and dest folder at the same time.\n')
-            exit(-1)
         else:
-            # has artifact_name and dest_dir, then download it
-            abs_dest_dir = os.path.abspath(self.options.dest_dir)
-            if os.path.exists(abs_dest_dir) and (not os.path.isdir(abs_dest_dir)):
-                print('### {} is not a folder.\n'.format(abs_dest_dir))
-                exit(-1)
+            # has artifact_name, then download it
+            if self.options.dest_dir:
```

Comment:
that's a great idea!

but I think we can just print a warning message and exit the program, due to this class is a program for user, not a library for developer.
IMO, the program for user should handle the exception, and library for developer should raise exception.
What do you think?